third_party/move/tools/move-resource-viewer/src/lib.rs

```diff
@@ -442,6 +442,10 @@ impl<V: CompiledModuleView> MoveValueAnnotator<V> {
             TypeTag::U256 => FatType::U256,
             TypeTag::U128 => FatType::U128,
             TypeTag::Vector(ty) => FatType::Vector(Box::new(self.resolve_type_impl(ty, limit)?)),
+            TypeTag::Function(..) => {
+                // TODO(#15664) implement functions for fat types"
```

Comment:
All these `todo`s are fine because function value resources are never on-chain, so this is unreachable, right?

Reply:
Nope, all these TODOs will need to be revisited, and the code is not production ready before each of them is resolved.

Reply:
I mean that if this code gets into release (as it will), it is not possible to reach this code and cause a panic